src/workers/parser/mapBindings.js

```diff
@@ -60,7 +60,7 @@ function globalizeDeclaration(node, bindings) {
       t.assignmentExpression(
         "=",
         getAssignmentTarget(declaration.id, bindings),
-        declaration.init
+        declaration.init || t.identifier("undefined")
```

Comment:
Would you be up for using `void 0` here so we don't risk picking up local declarations of variables named `undefined`, e.g. `var undefined = 4;` is fine, so could break this.

```
t.unaryExpression("void", t.numericLiteral(0))
```